fault/verilator_target.py

```diff
@@ -34,8 +41,21 @@ def flatten(l):
   Verilated::commandArgs(argc, argv);
   V{circuit_name}* top = new V{circuit_name};
 
+#if VM_TRACE
+  vluint64_t main_time = 0;
+
+  Verilated::traceEverOn(true);
+  tracer = new VerilatedVcdC;
+  top->trace(tracer, 99);
+  Verilated::mkdir("logs");
+  tracer->open("logs/vlt_dump.vcd");
```

Comment:
Also, let's change this to use the circuit name or verilog file name instead of the generic one, so we can distinguish between traces from different files.